github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -931,6 +931,34 @@ presubmits:
             memory: 4Gi
         securityContext:
           privileged: true
+  - always_run: false
```

I'd suggest to run it every time, so that we can collect data on this. But I'd also make in not required so that it doesn't block the merge. (It wouldn't anyway, since it doesn't report, but we might forget to set it later on)

```suggestion
  - always_run: true
    optional: false
```